catalog/task/update-infra-deployments/0.1/update-infra-deployments.yaml

```diff
@@ -0,0 +1,332 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "release"
+  name: update-infra-deployments
+spec:
+  description: |
+   This task clones redhat-appstudio/infra-deployments GH repository.
+   It then runs a script obtained from the infra-deployment-update-script key in extraData which can modify text files
+   It then generates pull-request for redhat-appstudio/infra-deployments repository using the modified files.
+  params:
+    - name: extraDataJsonPath
```

Comment:
I think this should have a default value, especially since we don't even let people control where `collect-data` stores it

Reply:
added default